src/common/createSlider.jsx

```diff
@@ -98,6 +98,7 @@ export default function createSlider(Component) {
         this.dragOffset = position - handlePosition;
         position = handlePosition;
       }
+      this.removeDocumentEvents();
```

Comment:
This will remove both TouchEvent and MouseEvent listener.

In mobile device, if there is no touchmove, the event will trigger follow `touchstart --> touchend --> onmousedown --> onmouseup`

Reply:
Eh..sorry. I am confused. Maybe I miss something. 
clear event before addEvent, there is `touchmove` again.  I don't know when that will happen.

Reply:
You mean I remove both TouchEvent and MouseEvent listener, so there is no more touchmove, the event will trigger follow `touchstart --> touchend --> onmousedown --> onmouseup`; 
But I just remove last event listener. In 103 lines, it add new event listener for this action. There is touchmove.

This bug happened when your mouse or touch move to another document(e.g. parent document with iframe).  The event had bind in origin document, can not be remove correctly. So we have to check with extra func, to ensure we have a clean context.

Or I misunderstood.  :D